### DISCUSSION TITLE
OpenZeppelin Package Error

### DISCUSSION THREAD

**Discussion by compot13** [Cyfrin/foundry-full-course-cu#2661] [Q&A] [answered]
Hello guys, 

I'm at (https://github.com/Cyfrin/foundry-defi-stablecoin-cu) and the OpenZeppelin Package doesn't contain security section for "ReentrancyGuard.sol".

Is there any updates I should make or did I miss something out?

Thanks in advance 🙏 

P.S. Snapshot below 
<img width="798" alt="Screenshot 2024-10-11 at 16 49 11" src="https://github.com/user-attachments/assets/a485e6d3-d557-4fd7-8319-fa3e9bc1d3cd">

**Comment by EngrPips** [comment, marked as answer]
Hello @compot13, Please use `cltr + p` (windows) or `command + p` (Mac) and type `ReentrancyGuard` in the search bar that appears in your `vscode.` let's see if the `ReentrancyGuard` file doesn't exist truly. See the way I searched below.

<img width="1440" alt="Screenshot 2024-10-11 at 15 19 10" src="https://github.com/user-attachments/assets/aaecdc0e-11c4-44b5-9182-930dbb47a161">

**Reply by compot13** [reply]
Yep, found it, the folder is "utils" not "security".

**Reply by EngrPips** [reply]
You are giga chad. Keep crushing it.
